source/guides/maintenance/custom-app-container.rst

```diff
@@ -127,300 +123,316 @@ OS version.
 
 .. note::
 
-		All of the higher level layers of multi-stage |CL| Dockerfiles inherit the
-		|CL| version from the base layer. You must rebuild the base layer(s) against the
-		desired OS version. In this example, 3 base layers must be rebuilt.
-
-
-#.	Rebuild the first base layer by adding an extra :command:`swupd_args`
-	argument to the command line. In the example below, the desired OS version is
-	30550.
+   All higher level layers of multi-stage |CL| Dockerfiles inherit the
+   |CL| version from the base layer. You must rebuild the base layers
+   against the desired OS version. In this example, 3 base layers must be
+   rebuilt.
 
-	.. code-block:: bash
-		:linenos:
-		:emphasize-lines: 6
+.. TODO: Add context similar to Example 1.
 
-		docker build \
-		--no-cache=true 
-		--build-arg http_proxy=$http_proxy \
-		--build-arg https_proxy=$https_proxy \
-		-t clearlinux/os-core:30550 \
-		--build-arg swupd_args="-m 30550" os-core/
+#. Rebuild the first base layer by adding an extra :command:`swupd_args`
+   argument to the command line. In the example below, the desired OS version is 30550.
 
-#.	Change the target Dockerfile to use the desired OS version. The example
-	below shows a diff of a modified clearlinux/redis Dockerfile that
-	uses the previously built clearlinux/os-core:30550 container image.
-
-	.. code-block:: diff
+   .. code-block:: bash
+      :linenos:
+      :emphasize-lines: 6
 
-		diff --git a/redis/Dockerfile b/redis/Dockerfile
-		index af977cb..d666e6e 100644
-		--- a/redis/Dockerfile
-		+++ b/redis/Dockerfile
-		@@ -7,7 +7,7 @@ RUN swupd update --no-boot-update $swupd_args
+      docker build \
+      --no-cache=true
+      --build-arg http_proxy=$http_proxy \
+      --build-arg https_proxy=$https_proxy \
+      -t clearlinux/os-core:30550 \
+      --build-arg swupd_args="-m 30550" os-core/
 
-		 # Grab os-release info from the minimal base image so
-		 # that the new content matches the exact OS version
-		-COPY --from=clearlinux/os-core:latest /usr/lib/os-release /
-		+COPY --from=clearlinux/os-core:30550 /usr/lib/os-release /
+#. Change the target Dockerfile to use the desired OS version. The example

```

Comment:
Change the target Dockerfile to use the desired OS version by changing the `latest` reference to the tag of the previously built version.